lib/averages.js

```diff
@@ -0,0 +1,57 @@
+var pry = require('pryjs');
+
+var total = function(numbers) {
+  return numbers.reduce((sum, number) => sum + number);
+}
+
+var mean = function(numbers) {
+  const sum = total(numbers);
+  return sum/numbers.length;
+}
+
+var median = function(array) {
+  let len = array.length 
+  if(len % 2 != 0) {
+    let index = Math.ceil(array.length/2 - 1);
```

Comment:
Any reason you didn't use Math.floor(array.length/2) to make a little shorter/more readable? Just something to consider